NEWS

```diff
@@ -1,6 +1,62 @@
 Release a.b
 -----------
 
+New work and changes:
+
+ * samtools import now outputs query grouped to the header.
```

Comment:
I know this is the title from the PR, but the language is a bit confusing if you don't understand ther SAM spec.

Maybe "samtools import now adds grouped by query-name to the SAM header"